Formula/c/codecov-cli.rb

```diff
@@ -0,0 +1,142 @@
+    sha256 "10294e9bf89cb713da05bc4790bdff616610432db561964827074898e174f917"
+  end
+
+  resource "regex" do
+    url "https://files.pythonhosted.org/packages/b5/39/31626e7e75b187fae7f121af3c538a991e725c744ac893cc2cfd70ce2853/regex-2023.12.25.tar.gz"
+    sha256 "29171aa128da69afdf4bde412d5bedc335f2ca8fcfe4489038577d05f16181e5"
+  end
+
+  resource "requests" do
+    url "https://files.pythonhosted.org/packages/9d/be/10918a2eac4ae9f02f6cfe6414b7a155ccd8f7f9d4380d62fd5b955065c3/requests-2.31.0.tar.gz"
+    sha256 "942c5a758f98d790eaed1a29cb6eefc7ffb0d1cf7af05c3d2791656dbd6ad1e1"
+  end
+
+  resource "responses" do
+    url "https://files.pythonhosted.org/packages/6d/db/b949a6bf2a75c64caea0a6b39d05e433aa2e51bea78ae9d5dda1110b31a5/responses-0.21.0.tar.gz"
+    sha256 "b82502eb5f09a0289d8e209e7bad71ef3978334f56d09b444253d5ad67bf5253"
+  end
+
+  resource "rfc3986" do
+    url "https://files.pythonhosted.org/packages/79/30/5b1b6c28c105629cc12b33bdcbb0b11b5bb1880c6cfbd955f9e792921aa8/rfc3986-1.5.0.tar.gz"
+    sha256 "270aaf10d87d0d4e095063c65bf3ddbc6ee3d0b226328ce21e036f946e421835"
+  end
+
+  resource "sniffio" do
+    url "https://files.pythonhosted.org/packages/cd/50/d49c388cae4ec10e8109b1b833fd265511840706808576df3ada99ecb0ac/sniffio-1.3.0.tar.gz"
+    sha256 "e60305c5e5d314f5389259b7f22aaa33d8f7dee49763119234af3755c55b9101"
+  end
+
+  resource "test-results-parser" do
+    url "https://files.pythonhosted.org/packages/fa/1f/35f5b23beb148c36285bfe9722adeaef976814670ebf9bd8129356a283c8/test_results_parser-0.1.0.tar.gz"
+    sha256 "0034281a4b406d7f072fc5ac1f5e44660e3c23bc92f2e7284862ee097f9626ee"
+  end
+
+  resource "tree-sitter" do
+    url "https://files.pythonhosted.org/packages/0d/52/1284e9ed195b161261ac09bfd9785027e2734fc77360a889f6464a8e8ce8/tree_sitter-0.20.2.tar.gz"
+    sha256 "0a6c06abaa55de174241a476b536173bba28241d2ea85d198d33aa8bf009f028"
+  end
+
+  resource "tree-sitter-javascript" do
+    url "https://github.com/tree-sitter/tree-sitter-javascript/archive/refs/tags/v0.20.2.tar.gz"
+    sha256 "5b9d8ae376053d7e6d2953aeb48edf7151d57d76b9a37c9c4590abec8195bafe"
+  end
+
+  resource "tree-sitter-python" do
+    url "https://github.com/tree-sitter/tree-sitter-python/archive/refs/tags/v0.20.2.tar.gz"
+    sha256 "fc8a893f6e08c7236054e43d0acc407ef757a9f47bbb9d910dbc56055d6b9ae6"
+  end
+
+  resource "urllib3" do
+    url "https://files.pythonhosted.org/packages/8b/00/db794bb94bf09cadb4ecd031c4295dd4e3536db4da958e20331d95f1edb7/urllib3-2.0.6.tar.gz"
+    sha256 "b19e1a85d206b56d7df1d5e683df4a7725252a964e3993648dd0fb5a1c157564"
+  end
+
+  resource "PyYAML" do
+    url "https://files.pythonhosted.org/packages/cd/e5/af35f7ea75cf72f2cd079c95ee16797de7cd71f29ea7c68ae5ce7be1eda0/PyYAML-6.0.1.tar.gz"
+    sha256 "bfdf460b1736c775f2ba9f6a92bca30bc2095067b8a9d77876d1fad6cc3b4a43"
+  end
+
+  def install
+    inreplace "setup.py", "codecovcli = ", "codecov = "
```

Comment:
Because Codecov package the binary themselves using that name.

<img width="653" alt="Codecov describing how to install a binary version, using `codecov` as the name of the binary executable." src="https://github.com/Homebrew/homebrew-core/assets/203366/1dc3469d-746c-4308-9f3f-8b563de4a1ca">


([source](https://github.com/codecov/codecov-cli/tree/v0.6.0?tab=readme-ov-file#as-a-binary))

Reply:
Context for my decision aside, you make a good point. I'll ask upstream to do that.

Reply:
Removed, thanks for the suggestion.